src/libecalc/presentation/exporter/formatters/formatter.py

```diff
@@ -1,73 +1,102 @@
 import abc
-from typing import Dict, List
+from datetime import datetime
+from typing import Dict, Iterator, List, Optional, Protocol, Tuple, Union
 
-from libecalc.presentation.exporter.dto.dtos import FilteredResult
+from libecalc.common.units import Unit
+
+ROW_ID = Union[str, int, float, datetime]
```

Comment:
got a bit confused with ALLCAPS here...is that some kind of convention? I would want to have `RowId` or `RowIndex` instead ... or? Same below